src/sentry/tasks/assemble.py

```diff
@@ -52,11 +51,7 @@ def assemble_dif(project_id, name, checksum, chunks, **kwargs):
                                     % len(result))
                 return
 
-            file_format, arch, file_id, filename, data = result[0]
-            dif, created = debugfile.create_dif_from_id(
-                project, file_format, arch, file_id, data,
-                os.path.basename(name),
```

Comment:
Note that this name override is now done inside `detect_dif_from_path`, including `basename()`.